docs/More.AndroidSupportStatus.md

```diff
@@ -58,7 +58,19 @@ All Core APIs are 100% implemented.
 ## Emulator control
 1. **Emulators** are fully supported, to choose an emulator to run your tests on check `emulator -list-avds`. If none exist, create one.
 2. **Devices** - Coming soon!
-3. **Genymotion** -  Coming a bit later...
```

Comment:
This does not relate to Genymotion specifically, it can be used with connected device, I think it should be emphesized in the docs.